### DISCUSSION TITLE
Allow user to specify 's' for the units

### DISCUSSION THREAD

**Discussion by nsolvepaul** [cytech/Home-Assistant-wundergroundpws#151] [Q&A] [answered]
Great integration
Would like ability to receive wind speed in m/s instead of kmh, looking at the API it suggests changing the units to 's' outputs windspeed in m/s
API link:
https://docs.google.com/document/d/13HTLgJDpsb39deFzk_YCQ5GoGoZCO_cRYzIxbwvgJLI/edit

**Comment by cytech** [comment, marked as answer]
for the weather entitity (hass weather card) you can change the windspeed unit in the card settings.
if you want this for the sensor.wupws_windspeed, you can create a template, something like:
km/h -> m/s
{{ (states('sensor.wupws_windspeed') |float/3.6)|round(2) }}

**Reply by nsolvepaul** [reply]
Great suggestion, thanks
